### ISSUE TITLE
Example issue

### ISSUE THREAD

**Issue by Psycokwet** [Psycokwet/Ask-me-anything#1]
Here I just created an issue to show you how it can look like :) 
Remember to write code, you can do
\```javascript
let coucou="bob"
\```
It will do the following : 
```javascript
let coucou="bob"
```
(How did I write the first one without having it showing it as code ? Well, I added an escape character "\\" it allows the next special character written to not have the effect it should have. It's so efficient I have to write twice \\ to show it only once, so I actually wrote "\\\\" -how many \\ did I wrote in the last one to show only two? 😄 )

**Comment by Dodoodilee** [comment]
you used it twice ! to show it only once . i guess

**Comment by Dodoodilee** [comment]
You Used 4 \ to display them as 2 in total :)

**Comment by Psycokwet** [comment]
Yes ! You understood very well the question this time, congratulations! 🎉

**Comment by Dodoodilee** [comment]
thank you so much for restructuring the question in a simple way !

**Comment by Psycokwet** [comment]
In the end I didn't have the time to edit the question ! So more kudos to you for being able yo re-read and understand better :)

Do not forget you can create other issues here so that we can talk about any code questions you have :)

**Comment by Dodoodilee** [comment]
what is kudos ?

**Comment by Psycokwet** [comment]
Kudos is something we say to congratulate someone :)

**Comment by Dodoodilee** [comment]
Thank You , i have learned a lot from you . Youre a blessing :)
